src/components/autocomplete/js/autocompleteController.js

```diff
@@ -466,9 +466,9 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
         select(ctrl.index);
         break;
       case $mdConstant.KEY_CODE.ESCAPE:
+        event.preventDefault(); // Prevent browser from always clearing input
```

Comment:
@topherfangio Yeah, you're right - it seems like Firefox doesn't have that behavior. It's showing up in Chrome.

Actually when having a `input[type="search"]`, the input will automatically clear its value on escape key.

It also strips line-breaks (See [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input))
```
line-breaks are automatically removed from the input value."
```

---
@clshortfuse Can you find a better solution?